jobs/ci-kubernetes-node-kubelet-conformance.sh

```diff
@@ -19,4 +19,23 @@ set -o pipefail
 set -o xtrace
 
 # PROJECT="k8s-jkns-ci-node-e2e"
-test/e2e_node/jenkins/conformance/conformance-jenkins.sh test/e2e_node/jenkins/conformance/jenkins-conformance.properties
```

Comment:
I'm not sure whether this one can use node runner. Because we need `docker load/save`, and I don't think we can access docker inside the container.